sdk/core/azure-core/src/main/java/com/azure/core/exception/package-info.java

```diff
@@ -1,7 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+
 /**
- * Package containing core exception classes.
+ * <p>This package contains the core exception classes used throughout the Azure SDKs.</p>
+ *
+ * <p>These exceptions are typically thrown in response to errors that occur when interacting with Azure services.
+ * For example, if a network request to an Azure service fails, an exception from this package might be thrown.
```

Comment:
```suggestion
 * For example, if a network request to an Azure service fails, an exception from this package is thrown.
```
It's possible that some other network request exception leaks out but the design is that we wrap exceptions so we can be more definitive here.